dvc/fs/base.py

```diff
@@ -184,6 +185,15 @@ def open(
     ) -> "IO":  # pylint: disable=arguments-differ
         return self.fs.open(path, mode=mode, encoding=encoding, **kwargs)
 
+    def read_block(
```

Comment:
Do we need all of these methods, if we are not using it anyway? Why not just inherit from AbstractFileSystem and get them for free?

Reply:
Maybe this one and others (`head`/`tail`/`sizes`/`du`) would work that way, but others require implementation. Since this is a wrapper, I prefer being explicit for now.

In the future, we may just use `__getattr__()` hack. :)